src/main/g8/build.sbt

```diff
@@ -1,5 +1,5 @@
-lazy val akkaHttpVersion = "$akka_http_version$"
-lazy val akkaVersion    = "$akka_version$"
+lazy val akkaHttpVersion = "10.0.11"
+lazy val akkaVersion    = "2.5.11"
```

Comment:
Pretty sure this change should not be done; it's a template after all -- those numbers are getting replaced during generation of project from template

Reply:
Sorry, it looked like the Travis CI build was failing on my pull request to fix a 404 error.